frontend/build/asset-manifest.json

```diff
@@ -1,18 +1,18 @@
 {
```

Comment:
Can we get all of `frontend/build/*` out of `master` (to prevent it from getting out of sync w/ the source) and into a `dist` (or something) branch?

Reply:
Yes, I'll do it!